config/settings.py

```diff
@@ -129,7 +134,7 @@
 
 # Parse database connection url strings like psql://user:pass@127.0.0.1:8458/db
 
-if env("LOCAL_RUN"):
+if DEBUG:
```

Comment:
Termina cu if DEBUG if not DEBUG. Care baza se foloseste trebuie definit in environment variables.
DEBUG=True nu inseamna ca e local iar DEBUG=false in container